avocado/utils/multipath.py

```diff
@@ -70,7 +70,7 @@ def device_exists(path):
     :return: True if path exists, False if does not exist.
     """
     cmd = "multipath -l %s" % path
-    if process.system(cmd, ignore_status=True, sudo=True) != 0:
+    if process.system(cmd, ignore_status=True, sudo=True, shell=True) != 0:
```

Comment:
` if process.system(cmd, ignore_status=True, sudo=True, shell=True):` should be sufficent